CMakeLists.txt

```diff
@@ -37,6 +37,9 @@ endif()
 set(CMAKE_CXX_STANDARD 14)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
+set(CMAKE_CXX_FLAGS "-g -O0 -Wall -fprofile-arcs -ftest-coverage")
```

Comment:
I'm not 100% sure but you can probably replace `-fprofile-arcs -ftest-coverage` with `--coverage`

https://stackoverflow.com/questions/28840261/gcov-what-is-the-difference-between-coverage-and-ftest-coverage-when-buildi